OrcScala/src/orc/API.scala

```diff
@@ -24,6 +24,12 @@ import orc.progress.ProgressMonitor
 import orc.values.Signal
 import scala.util.parsing.input.Position
 import orc.compile.CompilerFlagValue
+import orc.error.runtime.HaltException
+import orc.values.Field
+import orc.error.runtime.{ TypeNoSuchMemberException, UncallableTypeException, NoSuchMemberException, UncallableValueException, TypeDoesNotHaveMembersException }
+import orc.error.runtime.DoesNotHaveMembersException
+
+// TODO: This file is huge and covers many unrelated things. Split into multiple sub API with limited relationships. Also it's not clear where to put utility implementations/bases for these APIs.
```

Comment:
I agree. 

What about an Accessor.scala and an Invoker.scala which each include both the interfaces and the standard implementations? Or if you think it's better we could have two pairs, Accessor.scala and AccessorUtilities.scala, and then a similar pair for Invoker.